카펫.js

```diff
@@ -0,0 +1,6 @@
+function solution(brown, yellow) {
+    const board = brown + yellow;  
+    for (let i = parseInt(board / 2); i > 0; i--) {
```

Comment:
변수 선언 해준 뒤 조건문 들어가기 전 한줄 띄워주세요!
```suggestion
    const board = brown + yellow;  
    
    for (let i = parseInt(board / 2); i > 0; i--) {
```